packages/new-deploy/scripts/deploy.ts

```diff
@@ -0,0 +1,51 @@
+import DeployContext from '../src/DeployContext';
+import * as config from '@nomad-xyz/configuration';
+import * as ethers from 'ethers';
+import { NonceManager } from '@ethersproject/experimental';
+import fs from "fs";
+import * as dotenv from 'dotenv';
+import {getConfig,getOverrides} from "./utils";
+dotenv.config();
+
+run();
+
+/*
+* FLOW FOR ENGINEER TO ADD A NEW CHAIN:
+* 1. load published config to local file (package.json script)
```

Comment:
TODO: script to pull published config from URL and copy to local file?

Reply:
make issue

Reply:
made an issue, we won't block on it: https://github.com/nomad-xyz/monorepo/issues/110